tests/acceptance/features/apiDepthInfinity/propfind.feature

```diff
@@ -0,0 +1,114 @@
+      | path                                        |
+      | simple-folder                               |
+      | simple-folder/simple-folder1                |
+      | simple-folder/simple-empty-folder           |
+      | simple-folder/simple-folder1/simple-folder2 |
+    And user "Alice" has uploaded the following files with content "simple-test-content"
+      | path                                                      |
+      | textfile0.txt                                             |
+      | welcome.txt                                               |
+      | simple-folder/textfile0.txt                               |
+      | simple-folder/welcome.txt                                 |
+      | simple-folder/simple-folder1/textfile0.txt                |
+      | simple-folder/simple-folder1/welcome.txt                  |
+      | simple-folder/simple-folder1/simple-folder2/textfile0.txt |
+      | simple-folder/simple-folder1/simple-folder2/welcome.txt   |
+
+
+  Scenario Outline: get the list of resources with depth infinity
+    Given using <dav-path-version> DAV path
+    When user "Alice" lists the resources in "/" with depth "infinity" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the last DAV response for user "Alice" should contain these nodes
+      | name                                                      |
+      | textfile0.txt                                             |
+      | welcome.txt                                               |
+      | simple-folder/                                            |
+      | simple-folder/textfile0.txt                               |
+      | simple-folder/welcome.txt                                 |
+      | simple-folder/simple-empty-folder/                        |
+      | simple-folder/simple-folder1/                             |
+      | simple-folder/simple-folder1/simple-folder2               |
+      | simple-folder/simple-folder1/textfile0.txt                |
+      | simple-folder/simple-folder1/welcome.txt                  |
+      | simple-folder/simple-folder1/simple-folder2/textfile0.txt |
+      | simple-folder/simple-folder1/simple-folder2/welcome.txt   |
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
+
+
+  Scenario Outline: get the list of resources of a folder with depth infinity
+    Given using <dav-path-version> DAV path
+    When user "Alice" lists the resources in "/simple-folder" with depth "infinity" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the last DAV response for user "Alice" should contain these nodes
+      | name                                                      |
+      | /simple-folder/textfile0.txt                              |
+      | /simple-folder/welcome.txt                                |
+      | /simple-folder/simple-folder1/                            |
```

Comment:
Can we also remove these slashes if they're that relevant while asserting just to maintain consistency? 
```suggestion
      | simple-folder/textfile0.txt                              |
      | simple-folder/welcome.txt                                |
      | simple-folder/simple-folder1/                            |
```